content/07.pathogenesis.md

```diff
@@ -445,20 +448,20 @@ The extent to which aerosol transmission contributes to the spread of respirator
 In influenza A, for example, viral particles can be detected in aerosols produced by infected individuals, but it is not clear to what extent these particles drive the spread of influenza A infection [@pubmed:23771256; @doi:10.1097/QCO.0000000000000563; @doi:10.3201/eid1301.061202; @doi:10.1038/s41598-019-38825-y; @doi:10.1093/femsre/fuv039].
 Regardless of its role in the spread of influenza A, however, aerosol transmission likely played a role in outbreaks such as the 1918 Spanish Influenza (H1N1) and 2009 "swine flu" (pH1N1) [@doi:10.1093/femsre/fuv039].
 All three of these mechanisms have been identified as possible contributors to the transmission of HCoVs [@pubmed:23771256], including the highly pathogenic coronaviruses SARS-CoV-1 and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].
-Transmission of SARS-CoV-1 is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@pubmed:23771256; @pubmed:15631748; @doi:10.1111/j.1600-0668.2004.00317.x; @doi:10.1086/429634], and fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558].
+Transmission of SARS-CoV-1 is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@pubmed:23771256; @pubmed:15631748; @doi:10.1111/j.1600-0668.2004.00317.x; @doi:10.1086/429634], and fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558]. <!-- Were we able to get a translation of pubmed:15631748? -->
```

Comment:
```suggestion
Transmission of SARS-CoV-1 is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@pubmed:23771256; @doi:10.1111/j.1600-0668.2004.00317.x; @doi:10.1086/429634], and as suggested by the English abstract of an article from China [@pubmed:15631748].
Fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558].
```
I don't think we were able to find anyone who could read it. I might have a solution to this longer term, but for now perhaps we should leave it like this. I think that's why I had phrased the language pretty weakly ("considered possible")

Reply:
In that case, how about we remove `@pubmed:15631748` from that list of references and keep the phrasing as-is?  We already have three others to support the statement.

I also added a citation to https://greenelab.github.io/covid19-review/#ref-sUAAY1me (d5ed3a09abc2d742796355239a26febbe4ef2443) in the later paragraph where we discuss SARS-CoV-2 specifically.  We had already cited in in pharmaceuticals but not here.

Reply:
Thank you @agitter! This sounds good to me!

```suggestion
Transmission of SARS-CoV-1 is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@pubmed:23771256; @doi:10.1111/j.1600-0668.2004.00317.x; @doi:10.1086/429634], and fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558]. <!-- Were we able to get a translation of pubmed:15631748? -->
```